charts/k8s-monitoring/values.yaml

```diff
@@ -736,6 +712,37 @@ traces:
       # -- How long before sending
       timeout: 2s
 
+# Telemetry data receiver settings
+receivers:
+  grpc:
+    # -- Receive telemetry data over gRPC?
+    enabled: true
+    # -- Which port to use for the gRPC receiver. This port needs to be opened in the grafana-agent section below.
+    port: 4317
+    # -- It removes attributes which could cause high cardinality metrics. For example, attributes with IP addresses and port numbers in metrics about HTTP and gRPC connections will be removed.
+    disable_debug_metrics: true
+  http:
+    # -- Receive telemetry data over HTTP?
+    enabled: true
+    # -- Which port to use for the HTTP receiver. This port needs to be opened in the grafana-agent section below.
+    port: 4318
+    # -- It removes attributes which could cause high cardinality metrics. For example, attributes with IP addresses and port numbers in metrics about HTTP and gRPC connections will be removed.
+    disable_debug_metrics: true
+
+  prometheus:
+    # -- Receive Prometheus metrics
+    enabled: true
+    # -- Which port to use for the Prometheus receiver. This port needs to be opened in the grafana-agent section below.
+    port: 9999
+
+  zipkin:
+    # -- Receive Zipkin traces
+    enabled: true
```

Comment:
If it's a new config flag, should we disable by default as Zipkin instrumentation libraries' popularity is decreasing in favor of OpenTelemetry?

Reply:
Sounds good to me!